code/modules/mod/mod_core.dm

```diff
@@ -399,7 +399,7 @@
 	SIGNAL_HANDLER
 	if(mod.active)
 		particle_effect = new(mod.wearer, /particles/pollen, PARTICLE_ATTACH_MOB)
-		mob_spawner = mod.wearer.AddComponent(/datum/component/spawner, spawn_types=list(spawned_mob_type), spawn_time=5 SECONDS, max_spawned=3)
+		mob_spawner = mod.wearer.AddComponent(/datum/component/spawner, spawn_types=list(spawned_mob_type), spawn_time=5 SECONDS, max_spawned=3, faction=list(FACTION_NEUTRAL))
```

Comment:
what if im a nukie and my turrets shoot me because of butterflies

Reply:
clearly we need to copy the faction from the wearer